src/main/java/org/broadinstitute/hellbender/engine/GATKTool.java

```diff
@@ -157,7 +157,7 @@
      * Walker base classes (ReadWalker, etc.) are responsible for hooking these intervals up to
      * their particular driving data source.
      */
-    List<SimpleInterval> userIntervals;
```

Comment:
We would prefer not to expose these engine fields for safety purposes. As of right now these fields are only exposed to engine walkers etc... I would say revert this change.

Reply:
hmm... it looks like the tests/etc would be made somewhat easier if there were a getter for this. It should be harmless to add a `getUserIntervals()` that just returns an unmodifiable list of these simple intervals.